interface/patient_file/encounter/find_code_dynamic.php

```diff
@@ -0,0 +1,311 @@
+<?php
+/* Copyright (C) 2015-2017 Rod Roark <rod@sunsetsystems.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ */
+
+// For DataTables documentation see: http://legacy.datatables.net/
```

Comment:
See here for the new sleek license/author/copyright headers we are using:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#File-Level_DocBlock
(keep all the comments at the top of it)